### ISSUE TITLE
[A11Y] "show" link in the cart for bundle line items is not reachable

### ISSUE THREAD

**Issue by Larisa-Staroverova** [SAP/spartacus#11989]

**Comment by Larisa-Staroverova** [comment]
Known issue regarding [event.preventDefault()](https://stackoverflow.com/questions/33044607/if-i-set-a-break-point-in-keydown-event-the-key-i-typed-wont-be-appended-to-th)
